utility.css

```diff
@@ -313,6 +313,12 @@ small {
   font-weight: 600;
 }
 
+.font-weight-regular{
+ font-weight: 400;
+}
+.font-weight-light{
+  font-weight: 300;
+}
 .code-snippet {
   background-color: #151718;
```

Comment:
You can use variable here, if this seems reusable.

Reply:
Its only used at one place and it is not reusable, still I have changed it to color variable now